gordo_components/model/transformers/imputer.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+
+from typing import Union
+
+import pandas as pd
+import numpy as np
+
+from sklearn.base import TransformerMixin
+
+
+class InfImputer(TransformerMixin):
+    def __init__(
+        self,
+        inf_fill_value=None,
+        neg_inf_fill_value=None,
+        strategy="minmax",
+        power: float = 2.0,
+    ):
+        """
+        By default, fill -inf with lowest dtype value and inf with highest dtype value
+        of arrays. Otherwise specify fill values or a strategy.
+
+        Fill values (if not None) are given precidence over `strategy`
+
+        Parameters
+        ----------
+        inf_fill_value: numeric
+            Value to fill 'inf' values
+        neg_inf_fill_value: numeric
+            Value to fill '-inf' values
+        strategy: str
+            How to fill values, irrelevant if fill value is provided.
+            choices: 'extremes', 'minmax'
+            -'extremes' will use the min and max values for the current datatype.
+            such that 'inf' in a float32 dataset will have float32's largest value inserted.
+            - 'minmax' will look at the min and max values in the feature where the -inf / inf
+            appears and fill with the max/min found in that feature.
+        power: float
+            Only applicable if ``strategy='minmax'``
+            Will compound the max/min value by this power. If the max value in a feature was 10
+            and ``power=2`` any inf value will be filled with 20. Likewise, if the min feature
```

Comment:
What if the max is negative? Will the positive inf then be power*a_negative_number?